packages/rum/src/domain/rumEventsCollection/view/trackTimings.ts

```diff
@@ -10,26 +10,51 @@ export interface Timings {
 
 export function trackTimings(lifeCycle: LifeCycle, callback: (timings: Timings) => void) {
   let timings: Timings | undefined
-  const { unsubscribe: stopPerformanceTracking } = lifeCycle.subscribe(
+  function setTimings(newTimings: Partial<Timings>) {
+    timings = { ...timings, ...newTimings }
+    callback(timings)
+  }
+
+  const { stop: stopNavigationTracking } = trackNavigationTimings(lifeCycle, setTimings)
+  const { stop: stopFCPTracking } = trackFirstContentfulPaint(lifeCycle, (firstContentfulPaint) =>
+    setTimings({ firstContentfulPaint })
+  )
+
+  return {
+    stop() {
+      stopNavigationTracking()
+      stopFCPTracking()
+    },
+  }
+}
+
+function trackNavigationTimings(lifeCycle: LifeCycle, callback: (newTimings: Partial<Timings>) => void) {
+  const { unsubscribe: stop } = lifeCycle.subscribe(LifeCycleEventType.PERFORMANCE_ENTRY_COLLECTED, (entry) => {
```

Comment:
🤔 I am wondering if we should not rename `unsubscribe` into `stop` to be consistent with all the other stop methods